### DISCUSSION TITLE
add prop onerror to q-img

### DISCUSSION THREAD

**Discussion by darqus** [quasarframework/quasar#17369] [Ideas / Proposals] [answered]
```
<img
  :height="$q.screen.gt.md ? '172px' : '148px'"
  :src="course.course.icon?.thumbUrl || course.course.icon?.url"
  :width="$q.screen.gt.md ? '172px' : '148px'"
  onerror="this.src = '/lk/img/no-image.svg'"
/>
```
this work
![изображение](https://github.com/quasarframework/quasar/assets/30623895/b16a256e-77ff-47af-9e5c-83a329b416f1)

```
<q-img
  :height="$q.screen.gt.md ? '172px' : '148px'"
  :src="course.course.icon?.thumbUrl || course.course.icon?.url"
  :width="$q.screen.gt.md ? '172px' : '148px'"
  onerror="this.src = '/lk/img/no-image.svg'"
/>
```
this not work
![изображение](https://github.com/quasarframework/quasar/assets/30623895/e7116274-0c6d-4144-a066-0b4be9bd1484)

have to use img instead of q-img

**Comment by ivanjaros** [comment, marked as answer]
there already is
![obrázok](https://github.com/user-attachments/assets/6ababe90-b35c-412c-b442-a1645543606f)

**Reply by rstoenescu** [reply]
Furthermore, there already is the `error` event too. But events are attached like this: `<q-img @error="handler"` not `<q-img onerror="handler"`.
